jupyterlab/labbox_ephys_widgets_jp/labbox_ephys_widgets_jp/create_recording_view.py

```diff
@@ -48,9 +48,9 @@ def __init__(self) -> None:
             super().__init__()
             self.on_msg(self._handle_message)
             self._worker_session = le.WorkerSession(labbox_config=labbox_config)
-            def on_msg(msg):
-                self.send(msg)
-            self._worker_session.on_message(on_msg)
+            def on_msgs(msgs):
```

Comment:
Send lists of messages rather than individual messages from server->client. This should improve efficiency (avoid unnecessary interim re-renders on the client), but also allows sending groups of messages that must be processed together on the client.